src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32Program.cs

```diff
@@ -486,6 +507,46 @@ private static Win32Program ExeProgram(string path)
             }
         }
 
+        // Function to get the application type, given the path to the application
+        public static ApplicationType GetAppTypeFromPath(string path)
+        {
+            if (path == null)
+            {
+                throw new ArgumentNullException(nameof(path));
+            }
+
+            string extension = Extension(path);
+            ApplicationType appType = ApplicationType.GenericFile;
+
+            if (extension.Equals(ExeExtension, StringComparison.OrdinalIgnoreCase) || NonExeApplicationExtensions.Contains(extension))
```

Comment:
nit:  Do we need two separate lists?  Also maybe wrap this in a IsApplicationExtension method for clarity.

Reply:
Combined the two lists, and changed the parts of the code where only ExeExtension was referenced to use both. I don't think any of those code paths rely on the extension being .exe specifically and it should apply to .msc and other executable types as well.